app/views/layouts/_application_content.html.erb

```diff
@@ -2,19 +2,42 @@
   <div id="content" class="container">
     <%= notifications %>
     <% unless @page_header.blank? %>
-      <div class="row form-group">
-        <h1 class="col-md-8"><%=h @page_header %></h1>
-      </div>
+        <div class="row form-group">
+          <h1 class="col-md-8"><%=h @page_header %></h1>
+        </div>
+    <% end %>
+    <% if @count_nil_hosts != nil && @count_nil_hosts > 0 %>
+        <%= alert :class => 'alert-warning', :header => '',
+                  :text => (n_('There is', 'There are', @count_nil_hosts) + ' ' +
+                      link_to(n_("%{count} host with no %{taxonomy_single} assigned",
+                                 "%{count} hosts with no %{taxonomy_single} assigned", @count_nil_hosts) %
+                                  {:count => @count_nil_hosts , :taxonomy_single => taxonomy_single },
+                              hosts_path(:search => "not has #{controller_name.singularize}"))).html_safe %>
+    <% end %>
+    <% if searchable? && !(@page_header == "Audits" || @page_header == "Overview")  %>
+        <div class="row toolbar-pf table-view-pf-toolbar">
+          <div class="col-sm-12">
+            <div class="row">
+              <%= render "common/searchbar" if searchable? %>
+              <div id="title_action" class="col-md-6">
+                <div class="btn-toolbar pull-right"><%=h yield(:title_actions) %></div>
+              </div>
+            </div>
+            <% if @page_header == "Hosts" %>
+                <div class="row toolbar-pf-results">
+                  <%= results_info(:more => _("<b class='select_count'>0</b> of <b>#{@hosts.total_entries}</b> selected")) %>
+                </div>
+            <% end %>
+          </div>
+        </div>
+    <% else %>
```

Comment:
any way to simplify this `if` ... `else` ? it has duplicates, but we the order of the `<div>`'s have to be considered

Reply:
use a partial?